src/components/Locations.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import "../App.css";
+
+
+class Locations extends React.Component {
+    constructor() {
+        super()
+
+        this.state = {
+            location: []
+        }
+    }
+
+    fetchPeople = () => {
+        fetch('https://ghibliapi.herokuapp.com/location')
+        .then((res) => res.json())
+        .then((data) => {
+            this.setState({
+                location: data,
+            })
+        })
+    }
+
+    componentDidMount = () => {
+        this.fetchPeople()
+    }
```

Comment:
```suggestion
    componentDidMount = () => {
        this.fetchLocations()
    }
```
Again, `fetchPeople` is not a good function name in this context. It should be something like `fetchLocations`.